src/main-oe/sgxlkl_run_oe.c

```diff
@@ -767,7 +767,19 @@ void get_signed_libsgxlkl_path(char* path_buf, size_t len)
         base += strspn(base, ":");
     }
 
-    sgxlkl_host_fail("Unable to locate libsgxlkl.so.signed\n");
+    sgxlkl_host_fail("Unable to locate %s\n", libname);
+}
+
+/* Determines path of libsgxlkl.so.signed */
+void get_signed_libsgxlkl_path(char* path_buf, size_t len)
+{
+    find_lib("libsgxlkl.so.signed", path_buf, len);
+}
+
+/* Determines path of libsgxlkl-user */
+void get_libsgxlkl_user_path(char* path_buf, size_t len)
+{
+    find_lib("libsgxlkl-user.so", path_buf, len);
```

Comment:
For my understanding, is this shared object not signed like the other kernel counterpart?